.github/actions/run-integration-tests/action.yml

```diff
@@ -23,6 +23,9 @@ inputs:
   gadget_tag:
     description: 'Tag used for gadgets OCI images'
     required: true
+  gadget_verify_image:
+    description: 'Whether to verify the gadget image.'
+    default: 'true'
```

Comment:
Why is it needed? Do we want forks to be able to customize it?

Reply:
Yeah, like if they configure their own secrets they are still able to verify images!

Reply:
Can't we hardcode it to always be true? (If they're really into it they could change the hardcoded value from the makefile).

Reply:
We chatted internally, I've totally missed that this is set to `gadget_verify_image: ${{ needs.check-secrets.outputs.cosign }}` on the workflow, so my idea of hardcoding it doesn't make sense.